packaging/ubuntu-16.04/copyright

```diff
@@ -6,7 +6,7 @@ Files: *
 Copyright: Copyright (C) 2014,2015 Canonical, Ltd.
 License: GPL-3
  This program is free software: you can redistribute it and/or modify it
- under the terms of the the GNU General Public License version 3, as
```

Comment:
huh I wonder how this typo crept in. I'm slightly hesitant to change this instance since it is part of the license so maybe it's important to have the exact text, etc. but this is probably fine. I will defer to others whether this is a real concern or not.

Reply:
I think this is fine as we are not changing the license. We are changing the machine-readable document that says what the license is. The license stays intact.